conf.go

```diff
@@ -32,6 +32,46 @@ type Sourcer interface {
 	Source(fld Field) (string, bool)
 }
 
+const versionKey = `SVN`
```

Comment:
It seems only Details uses these constants, so move them where they are being used.

Also, all comments must be proper sentences with punctuation.